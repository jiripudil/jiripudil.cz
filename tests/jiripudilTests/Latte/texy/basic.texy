## Title of the document

Some paragraph.

- some
- list
- items

And another paragraph.
