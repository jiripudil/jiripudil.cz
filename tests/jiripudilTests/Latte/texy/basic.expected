
<h2>Title of the document</h2>

<p>Some paragraph.</p>

<ul>
	<li>some</li>

	<li>list</li>

	<li>items</li>
</ul>

<p>And another paragraph.</p>
